applications/new_schema.py

```diff
@@ -0,0 +1,75 @@
+import django_filters
+import graphene
+from graphene_django.filter import DjangoFilterConnectionField
+from graphene_django.types import DjangoObjectType
+from graphql_jwt.decorators import login_required, superuser_required
+
+from .models import BerthApplication, BerthSwitch, HarborChoice
+from .schema import BerthSwitchType as OldBerthSwitchType
+
+
+class HarborChoiceType(DjangoObjectType):
+    harbor = graphene.String()
+
+    class Meta:
+        model = HarborChoice
+        exclude = ("id", "application")
+
+    def resolve_harbor(self, info, **kwargs):
+        if self.harbor:
+            return self.harbor.servicemap_id
+        return None
+
+
+class BerthSwitchType(OldBerthSwitchType):
+    harbor = graphene.String()
+
+    class Meta:
+        model = BerthSwitch
+
+    def resolve_harbor(self, info, **kwargs):
+        if self.harbor:
+            return self.harbor.servicemap_id
+        return None
+
+
+class BerthApplicationFilter(django_filters.FilterSet):
+    has_berth_switch = django_filters.BooleanFilter(
```

Comment:
this is a bit of a technical way to name it, could you make it just `switch_application`? 🙂

Reply:
and even better: `switch_applications` 🙂